diff --git a/urdf/RH5.urdf b/urdf/RH5.urdf
index d814df5..d379524 100644
--- a/urdf/RH5.urdf
+++ b/urdf/RH5.urdf
@@ -1277,66 +1277,6 @@
       </geometry>
     </collision>
   </link>
-  <joint name="GLF2_DistanceSensorDistal_1" type="fixed">
-    <origin xyz="0.000299999833291532 -0.0247775082611843 -0.0163492159542979" rpy="0.0 1.13794911843156 1.5707963267949"/>
-    <parent link="GLF2TopSegment_Link"/>
-    <child link="GLF2_DistanceSensorDistal_1_Link"/>
-  </joint>
-  <link name="GLF2_DistanceSensorDistal_1_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <joint name="GLF2_DistanceSensorDistal_2" type="fixed">
-    <origin xyz="0.000199995310425429 0.0507125844075086 -0.0195804953991652" rpy="3.14159265358977 -0.751361283639969 1.57079632679489"/>
-    <parent link="GLF2TopSegment_Link"/>
-    <child link="GLF2_DistanceSensorDistal_2_Link"/>
-  </joint>
-  <link name="GLF2_DistanceSensorDistal_2_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <joint name="GLF2_TouchSensorDistal" type="fixed">
-    <origin xyz="0.000200002939819318 0.0407117537406957 -0.0292563156876496" rpy="-3.14159265358978 1.17417004512815 -1.57079632679488"/>
-    <parent link="GLF2TopSegment_Link"/>
-    <child link="GLF2_TouchSensorDistal_Link"/>
-  </joint>
-  <link name="GLF2_TouchSensorDistal_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <joint name="GLF2_TouchSensorIntermedial" type="fixed">
-    <origin xyz="0.000200000530702645 0.0123371368999994 -0.01766647971245" rpy="-3.14159265358978 1.18769356276747 -1.57079632679488"/>
-    <parent link="GLF2TopSegment_Link"/>
-    <child link="GLF2_TouchSensorIntermedial_Link"/>
-  </joint>
-  <link name="GLF2_TouchSensorIntermedial_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <joint name="GLF2_TouchSensor_FingerProximal" type="fixed">
-    <origin xyz="0.0003 0.0088011 0.024649" rpy="0.0 -0.15059 1.5708"/>
-    <parent link="GLF2ProximalSegment_Link"/>
-    <child link="GLF2_TouchSensor_FingerProximal_Link"/>
-  </joint>
-  <link name="GLF2_TouchSensor_FingerProximal_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
   <joint name="GLF3Base" type="fixed">
     <origin xyz="-0.026 0.0 0.0" rpy="0.0 0.0 3.1415926"/>
     <parent link="Connector_GLF3_Link"/>
@@ -1473,66 +1413,6 @@
       </geometry>
     </collision>
   </link>
-  <joint name="GLF3_DistanceSensorDistal_1" type="fixed">
-    <origin xyz="0.000299999833291532 -0.0247775082611843 -0.0163492159542979" rpy="0.0 1.13794911843156 1.5707963267949"/>
-    <parent link="GLF3TopSegment_Link"/>
-    <child link="GLF3_DistanceSensorDistal_1_Link"/>
-  </joint>
-  <link name="GLF3_DistanceSensorDistal_1_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <joint name="GLF3_DistanceSensorDistal_2" type="fixed">
-    <origin xyz="0.000199995310425429 0.0507125844075086 -0.0195804953991652" rpy="3.14159265358977 -0.751361283639969 1.57079632679489"/>
-    <parent link="GLF3TopSegment_Link"/>
-    <child link="GLF3_DistanceSensorDistal_2_Link"/>
-  </joint>
-  <link name="GLF3_DistanceSensorDistal_2_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <joint name="GLF3_TouchSensorDistal" type="fixed">
-    <origin xyz="0.000200002939819318 0.0407117537406957 -0.0292563156876496" rpy="-3.14159265358978 1.17417004512815 -1.57079632679488"/>
-    <parent link="GLF3TopSegment_Link"/>
-    <child link="GLF3_TouchSensorDistal_Link"/>
-  </joint>
-  <link name="GLF3_TouchSensorDistal_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <joint name="GLF3_TouchSensorIntermedial" type="fixed">
-    <origin xyz="0.000200000530702645 0.0123371368999994 -0.01766647971245" rpy="-3.14159265358978 1.18769356276747 -1.57079632679488"/>
-    <parent link="GLF3TopSegment_Link"/>
-    <child link="GLF3_TouchSensorIntermedial_Link"/>
-  </joint>
-  <link name="GLF3_TouchSensorIntermedial_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <joint name="GLF3_TouchSensor_FingerProximal" type="fixed">
-    <origin xyz="0.0003 0.0088011 0.024649" rpy="0.0 -0.15059 1.5708"/>
-    <parent link="GLF3ProximalSegment_Link"/>
-    <child link="GLF3_TouchSensor_FingerProximal_Link"/>
-  </joint>
-  <link name="GLF3_TouchSensor_FingerProximal_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
   <joint name="GLF4Base" type="fixed">
     <origin xyz="-0.026 0.0 0.0" rpy="0.0 0.0 3.1415926"/>
     <parent link="Connector_GL_T_Finger_Link"/>
@@ -1756,78 +1636,6 @@
       </geometry>
     </collision>
   </link>
-  <joint name="GL_DistanceSensor1" type="fixed">
-    <origin xyz="-0.000863 0.037716 0.083475" rpy="3.141593 -1.570793 3.141593"/>
-    <parent link="GL_Base_Link"/>
-    <child link="GL_DistanceSensor1_Link"/>
-  </joint>
-  <joint name="GL_DistanceSensor1_2" type="fixed">
-    <origin xyz="-0.000863 -0.037716 0.083475" rpy="0.0 -1.5708 0.0"/>
-    <parent link="GR_Base_Link"/>
-    <child link="GL_DistanceSensor1_Link_2"/>
-  </joint>
-  <link name="GL_DistanceSensor1_Link">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <link name="GL_DistanceSensor1_Link_2">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <joint name="GL_DistanceSensor2" type="fixed">
-    <origin xyz="-0.000709 -0.002977 0.083475" rpy="3.141593 -1.570793 3.141593"/>
-    <parent link="GL_Base_Link"/>
-    <child link="GL_DistanceSensor2_Link"/>
-  </joint>
-  <joint name="GL_DistanceSensor2_2" type="fixed">
-    <origin xyz="-0.0007089 0.0029774 0.083475" rpy="0.0 -1.5708 0.0"/>
-    <parent link="GR_Base_Link"/>
-    <child link="GL_DistanceSensor2_Link_2"/>
-  </joint>
-  <link name="GL_DistanceSensor2_Link">
-    <inertial>
-      <origin xyz="0.0 -0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <link name="GL_DistanceSensor2_Link_2">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <joint name="GL_IMU" type="fixed">
-    <origin xyz="-0.003256 0.00202 0.053849" rpy="0.0 1.3526 0.0"/>
-    <parent link="GL_Base_Link"/>
-    <child link="GL_IMU_Link"/>
-  </joint>
-  <joint name="GL_IMU_2" type="fixed">
-    <origin xyz="-0.0032559 -0.0020197 0.053849" rpy="0.0 1.3526 0.0"/>
-    <parent link="GR_Base_Link"/>
-    <child link="GL_IMU_Link_2"/>
-  </joint>
-  <link name="GL_IMU_Link">
-    <inertial>
-      <origin xyz="0.0 -0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
-  <link name="GL_IMU_Link_2">
-    <inertial>
-      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
-      <mass value="1e-05"/>
-      <inertia ixx="1e-05" ixy="0.0" ixz="0.0" iyy="1e-05" iyz="0.0" izz="1e-05"/>
-    </inertial>
-  </link>
   <joint name="GL_Thumb" type="fixed">
     <origin xyz="0.019161 -0.002327 0.042" rpy="0.0 0.0 3.141593"/>
     <parent link="GL_Base_Link"/>
